tools/docker/Dockerfile.monorepo

```
@@ -56,6 +56,7 @@ RUN --mount=type=cache,target=/var/lib/apt/lists/,sharing=private \
         "php${PHP_VERSION}-mbstring" \
         "php${PHP_VERSION}-mysqli" \
         "php${PHP_VERSION}-pcov" \
+        "php${PHP_VERSION}-sqlite" \
```

Somehow this works (when I try it in a local container there's a message about "Note, selecting 'php8.2-sqlite3' for regex 'php8.2-sqlite'"), but the actual package name is "sqlite3".
```suggestion
        "php${PHP_VERSION}-sqlite3" \
```